extensions/wikia/AdEngine/js/video/googleIma.js

```diff
@@ -153,18 +163,25 @@ define('ext.wikia.adEngine.video.googleIma', [
 	function setupIma(vastUrl, adContainer, width, height) {
 		var ima = createIma();
 
+		console.log('**** SETUP IMA');
+
 		function adsManagerLoadedCallback(adsManagerLoadedEvent){
+			console.log('*******ADS MANAGER LOADED CALLBACK');
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this logging statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS2228)